src/var.c

```diff
@@ -41,20 +41,18 @@
 #include <rdma/fi_var.h>
 #include <rdma/fi_log.h>
 
-struct fi_var_t {
-	const struct fi_provider *provider;
-	char *var_name;
-	char *help_string;
-
-	char *env_var_name;
-
-	struct fi_var_t *next;
-};
+#include "fi.h"
 
 static struct fi_var_t *fi_vars = NULL;
 
+__attribute__((visibility ("default")))
+struct fi_var_t *DEFAULT_SYMVER_PRE(_fi_getvars)(void) {
```

Comment:
Yes probably. This is a public facing API but it should only be used by tooling or otherwise "special" users. Not sure if there is a way we can communicate that, or maybe we don't.